resources/webroot/login.mustache

```diff
@@ -12,7 +12,7 @@
         <form action="/login" method="post">
             <div class="form-group">
                 <label for="username-email">{{login_username_email}}</label>
-                <input type="username-email" class="form-control" id="username-email" name="username-email" value="{{username-email}}" required>
+                <input type="username" class="form-control" id="username-email" name="username-email" value="{{username-email}}" required>
```

Comment:
afaik input type="username" is not a valid type
```
<input type="email" class="form-control" id="username-email" name="username-email" value="{{username-email}}" pattern="/^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/" required>
```
pattern makes it require having a valid email address

Reply:
It doesn’t need to be an email. Can be an username also

Reply:
So the type can be "email" or "username", because both is supported to login. But a password manager (in my case) cannot handle the type "username-email", we have to decide :)